### PR TITLE
Add the queries to make it responsive

### PR DESCRIPTION
**Pull request by kdaglas**
Ive made the website responsive so that it can be viewed on most screens

### CODE REVIEW

**Comment by 66hakeem** [comment]
Looks great @kdaglas

**Comment by SokoPaulSokool** [comment]
great work. maybe you can also host the host the site on gh-pages

File: css/main.css [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+a {
+    text-decoration: none;
+    color: #c0392b;
+}
+
+a:hover {
+    color: #e74c3c;
+}
+
```
Comment (66hakeem) [inline css/main.css:16]: Not a big issue your css got lots of white space

Comment (kdaglas) [reply css/main.css:16]: my visual studio code has this pep8 kind of linter that does that to the code

**Comment by alexkayabula** [comment]
Nice work

**Pull request merged**